data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
 	on offer
 		set "license: Coalition"
 		fail
+
+
+
+mission "Kimek Student 1"
+	minor
+	name "Transport Chikee"
+	description "Chikee, a young Kimek, is studying to become a Heliarch. Transport him to <destination>, where he was offered a job, so that he can make a living while studying on his own."
+	passengers 1
+	to offer
+		has "Coalition: First Contact: done"
+	source "Fourth Shadow"
+	destination "Second Rose"
+	on offer
+		conversation
+			`At any given time there are thousands of Kimek here in the spaceport, either mingling among themselves during a break from work, or negotiating travel prices with starship captains. You imagine that if you weren't a human and clearly stood out, you too would be constantly getting offers to travel to another world, as many civilian captains here wish to fill a last bunk or two they have vacant on their ship before departing.`
+			`	You manage to find a restaurant here where you spot some food you've already learned you can eat, so you take a seat and order something.`
+			`	You're halfway through your meal when you notice the waiter who brought it to you still hasn't moved away from the table.`
+			choice
+				`	"Can I help you?"`
+				`	"Sorry, should I have paid first before eating?"`
+			branch translator
+				has "language: Coalition"
+			`	The Kimek doesn't have a translation box, so your words don't reach him, and while he insists in remaining by the table, you also notice he has been constantly glancing at the entrance, as if waiting for someone to come in.`
+			`	When a Heliarch agent passes by the entrance, the Kimek says something in his language, heading to meet the Heliarch, and speaking with them briefly. The Heliarch agent then accompanies the Kimek waiter to your table.`
+			choice
+				`	"Is there an officer problem?"`
```

Comment:
Is there a problem officer (Or is it intentional?)

Reply:
you could suggest a commit, you know. makes it easier on him.

Reply:
was intentional as a joke